modules/administration/pages/ansible-setup-control-node.adoc

```diff
@@ -25,6 +27,16 @@ The {productname} client tools contain the [package]``ansible`` package.
 
 . Select the menu:Events[] tab and verify the status of the highstate.
 
+[NOTE]
+====
+If you want to install a newer Ansible on a {sle} 15 SP4 or SP5 client, you must enable only the [literal]``Python 3 Module``.
+====
+
+[NOTE]
+====
+With newer Ansible, systems with an old Python version cannot be managed anymore.
+====
```

Comment:
We can probably extend it with something like this 

Newer versions of Ansible no longer support managing nodes with outdated Python versions 
If a managed node still defaults to an older Python version, you may encounter connection errors or failures during playbook runs. To address this, user should  Upgrade Python on the managed node, if possible and Set the correct Python interpreter in your Ansible inventory or configuration.

Reply:
@meaksh can you please verify this

Reply:
that's correct :+1: